src/Core/WinSWCore/LogAppenders.cs

```diff
@@ -328,13 +329,17 @@ public class RollingSizeTimeLogAppender : AbstractFileLogAppender
         public int SizeTheshold { private set; get; }
         public string FilePattern { private set; get; }
         public TimeSpan? AutoRollAtTime { private set; get; }
+        public int? ZipOlderTthanNumDays { private set; get; }

```

Comment:
Was it supposed to be `ZipOlderThanNumDays`?

Reply:
Fixed